src/main/java/com/google/sps/data/EntryObject.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EntryObject {
+  List<Object> hourList = new ArrayList<>();
+  List<Object> difficultyList = new ArrayList<>();
+  List<String> container = new ArrayList();
```

Comment:
You initialize container here but also use a different container variable inside the different functions. Would change the name if it's necessary to have it initialized outside.